update_wow_addons.py

```diff
@@ -19,10 +19,8 @@ def __init__(self, testing=False):
         self.testing = testing  # if true, game dir changes and random addons are updated
 
         self.os = pf_system()
-        self.windows = self.os == 'Windows'
-        self.linux = self.os == 'Linux'
 
-        if not self.windows and not self.linux:
+        if not self.os in ["Darwin", "Windows","Linux"]:
```

Comment:
For stylistic consistency (and my OCD) please replace these double quotes with single quotes

Reply:
Done, made the not <value> in a <value> not in too :)